bin/configs/php-flight.yaml

```diff
@@ -0,0 +1,6 @@
+generatorName: php-flight
+outputDir: samples/server/petstore/php-flight
```

Comment:
thanks for the PR.

can you please add the new output folder to `.github/workflows/samples-php8.yaml` so that CI will test it moving forward?